packages/ic-http-certification/src/hash/request_hash.rs

```diff
@@ -37,14 +35,14 @@ pub fn request_hash(
 
 fn get_filtered_headers(
     headers: &[(String, String)],
-    request_certification: &RequestCertification,
+    request_certification: &DefaultRequestCertification,
 ) -> Vec<(String, Value)> {
     headers
         .iter()
         .filter_map(|(header_name, header_value)| {
             let is_header_included =
                 request_certification
-                    .certified_request_headers
+                    .headers
```

Comment:
any reason for the change here, wouldn't this now include headers that shouldn't go through certification?